theSrc/scripts/LabeledScatter.js

```diff
@@ -177,10 +176,25 @@ class LabeledScatter {
           await FitLine.draw(this.rootElement, config)
         }
         if (config.quadrantsShow) {
-          const original_x_range = _.cloneDeep(plotlyChart._fullLayout.xaxis.range)
-          const original_y_range = _.cloneDeep(plotlyChart._fullLayout.yaxis.range)
+          // Save range to prevent autorange from adding gap
+          // between plot border to quadrant background color
+          const original_x_min = plotlyChart._fullLayout.xaxis.range[0]
+          const original_x_max = plotlyChart._fullLayout.xaxis.range[1]
+          const original_y_min = plotlyChart._fullLayout.yaxis.range[0]
+          const original_y_max = plotlyChart._fullLayout.yaxis.range[1]
+
```

Comment:
I know this looks strange, but for some reason even when I use cloneDeep, the range would sometimes update after zooming, which made double click stop working. But split up like this, it works.